apps/hoarder/docker-compose.yml

```diff
@@ -0,0 +1,61 @@
+services:
+  web:
+    image: ghcr.io/hoarder-app/hoarder:0.16.0
```

Comment:
The image version should be the exact same with the `version` field in `config.json`